bundles/org.openhab.binding.renault/README.md

```diff
@@ -69,18 +72,21 @@ renaultcar.sitemap:
 sitemap renaultcar label="Renault Car" {
     Frame {
         Image item=RenaultCar_ImageURL
-        Default item=RenaultCar_BatteryLevel icon="batterylevel"
-        Default item=RenaultCar_BatteryEnergyAvailable icon="energy"
-        Default item=RenaultCar_PlugStatus icon="poweroutlet"
-        Default item=RenaultCar_ChargingStatus icon="switch"
-        Selection item=RenaultCar_ChargingMode mappings=[SCHEDULE_MODE="Schedule mode",ALWAYS_CHARGING="Instant charge"] icon="switch"
-        Default item=RenaultCar_ChargingTimeRemaining icon="time"
-        Default item=RenaultCar_EstimatedRange
-        Default item=RenaultCar_Odometer
-        Selection item=RenaultCar_HVACStatus mappings=[ON="ON"] icon="switch"
-        Setpoint item=RenaultCar_HVACTargetTemperature minValue=19 maxValue=21 step=1 icon="temperature"
-        Default item=RenaultCar_LocationUpdate icon="time"
-        Default item=RenaultCar_Location
+        Switch icon="switch" item=RenaultCar_UpdateNow
+        Default icon="batterylevel" item=RenaultCar_BatteryLevel
+        Default icon="energy" item=RenaultCar_BatteryEnergyAvailable
+        Default icon="time" item=RenaultCar_BatteryStatusUpdated
```

Comment:
icon wouldn't be needed here if it was added to the channel type, since created items would then automatically inherit it.